Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -196,7 +193,25 @@ public IncrementalLoadingCollection(TSource source, int itemsPerPage = 20, Actio
         /// An object of the <see cref="LoadMoreItemsAsync(uint)"/> that specifies how many items have been actually retrieved.
         /// </returns>
         public IAsyncOperation<LoadMoreItemsResult> LoadMoreItemsAsync(uint count)
-            => AsyncInfo.Run((c) => LoadMoreItemsAsync(count, c));
+            => LoadMoreItemsAsync(count, new CancellationToken(false)).AsAsyncOperation();
+
+        /// <summary>
+        /// Clears the collection and reloads data from the source
+        /// </summary>
+        public void Refresh()
+        {
+            if (this.IsLoading)
+            {
+                _refreshOnLoad = true;
+            }
+            else
+            {
+                Clear();
+                CurrentPageIndex = 0;
+                HasMoreItems = true;
+                DontAwait(LoadMoreItemsAsync(1));
```

Comment:
I agree with you @deltakosh.

@ianier I'll give it a test but wouldn't just clearing the collection, settings the index to 0 and HasMoreItems to true cause it to refresh anyway due to the way the collection works?